README.md

```diff
@@ -1,2 +1,53 @@
-# dgi_ir_customizations
-Implementation customizations forked from the original dgi_ir project.
+# DGI IR Customizations
+
+## Introduction
+
+Modules and functionality added to Discoverygarden's Institutional Repository
+implementation.
+
+**N.B.** Functionality here is branched from the original `dgi_ir` project;
+pieces of that module migrated to here were considered integral to DGI's
+Institutional Repository implementation rather than integral to Research Output
+itself. As such, this module _must be left public_.
```

Comment:
Not... sure I understand this assertion, that it must be left public?

Reply:
just trying to be transparent for the eventual publicization of this, like ... we had a lot of functionality that was baked into our theme stuff, which is being removed from the main module; i don't want to get into a situation where we're just removing things and making them private and not communicating what's going on

Reply:
> not communicating

... we're not rebasing and removing things from the history... the PRs are there... should the PRs have more verbose messages perhaps? Or _do_ we intend to rebase and or snapshot the repo and push that? I feel like I'm still missing something here.